itou/users/admin.py

```diff
@@ -90,6 +101,16 @@ class ItouUserAdmin(UserAdmin):
         ),
     )
 
+    def has_verified_email(self, obj):
+        """
+        Quickly indentify unverified email that could be the result of a typo.
```

Comment:
Typo `s/identify/identify/`

Reply:
Ironic in a comment mentioning a typo :-D

Reply:
Recursif